st2api/st2api/controllers/v1/pack_configs.py

```diff
@@ -0,0 +1,106 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pecan
+
+from st2common.constants.keyvalue import USER_SCOPE
+from st2common.services import packs as packs_service
+from st2common.models.api.base import jsexpose
+from st2api.controllers.resource import ResourceController
+from st2common.models.api.pack import ConfigAPI
+from st2common.models.api.pack import ConfigItemSetAPI
+from st2common.persistence.pack import Config
+from st2common.services.config import set_datastore_value_for_config_key
+from st2common.rbac.utils import get_user_db_from_request
+from st2common.rbac.types import PermissionType
+from st2common.rbac.decorators import request_user_has_permission
+from st2common.rbac.decorators import request_user_has_resource_db_permission
+
+__all__ = [
+    'PackConfigsController'
+]
+
+
+class PackConfigsController(ResourceController):
```

Comment:
Yes, but since this is a top-level controller (I wish it wasn't, but we are all aware of our pecan limitations right now...), same reason as here applies - https://github.com/StackStorm/st2/pull/2708#discussion_r64530912